Makefile

```diff
@@ -300,6 +300,8 @@ install-cri-deps: $(BINARIES)
 	@$(INSTALL) -D -m 755 bin/* ${CRIDIR}/usr/local/bin
 	@$(INSTALL) -d ${CRIDIR}/opt/containerd/cluster
 	@cp -r contrib/gce ${CRIDIR}/opt/containerd/cluster/
+	@$(INSTALL) -d ${CRIDIR}/etc/modules-load.d
+	@$(INSTALL) -m 644 containerd-modules-load.conf ${CRIDIR}/etc/modules-load.d/container.conf
```

Comment:
I'm not sure this is the right thing to do; this loads the module at boot time, which means that a reboot would be needed after installation of containerd ? https://www.freedesktop.org/software/systemd/man/modules-load.d.html

I'm also not sure if we should modify a system-wide configuration, where dynamically loading the module is more common; the man-page itself even is not recommending to use it;

> Note that it is usually a better idea to rely on the automatic module loading by PCI IDs, USB IDs, DMI IDs or similar triggers encoded in the kernel modules themselves instead of static configuration like this. In fact, most modern kernel modules are prepared for automatic loading already.

Reply:
I added lines 303 and 304, because the `containerd.service` file in line 306 does not contain the `modprobe` call anymore and the integration tests should not fail.

> I'm not sure this is the right thing to do; this loads the module at boot time, which means that a reboot would be needed after installation of containerd ? https://www.freedesktop.org/software/systemd/man/modules-load.d.html

Not really. The package maintainer could add a `modprobe overlay` call at the postinst-script.

> I'm also not sure if we should modify a system-wide configuration, where dynamically loading the module is more common; the man-page itself even is not recommending to use it;

Current behavior:
- There is currently no "automatic" loading of the kernel module either, instead there is an explicit call to `modprobe overlay`.
- There is already a system-wide loading of the module, since the `modprobe` call is stored in the `systemd.service` file and, I now assume, this is "enabled" and thus also executed at system startup.

New behavior:
- There is also an explicit loading of the kernel module, timed just a bit earlier in the boot process -> no change to the current behavior.
- Administrators can thus find kernel module calls that absolutely have to be loaded in a central location.

Remark to the automatic kernel module loading:
- An automatic loading of a kernel module only works / can only work, if e.g. using device files is intercepted by udev rules and then the needed module is loaded, what seems to be impossible with the `overlay` kernel module unfortunately. Anyway, I have not found anything about this.

Reply:
> Administrators can thus find kernel module calls that absolutely have to be loaded in a central location.

So, it's not absolutely needed, as containerd supports other storage drivers (devicemapper, btrfs, zfs).

Reply:
I do not understand your comment. Or rather, I don't know what you want to express with it.